controller/clientController.ts

```diff
@@ -0,0 +1,77 @@
+import { Application, Request, Response } from "express";
+import { AssignClient } from "../model/assignClient";
+import { Client } from "../model/client";
```

Comment:
If you are importing something just for typing, you should use import type. Doing this means the import is stripped out during the build resulting in a smaller build package